packages/php/blueprint/src/Importers/ImportSetSiteOptions.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Automattic\WooCommerce\Blueprint\Importers;
+
+use Automattic\WooCommerce\Blueprint\StepProcessor;
+use Automattic\WooCommerce\Blueprint\StepProcessorResult;
+use Automattic\WooCommerce\Blueprint\Steps\SetSiteOptions;
+use Automattic\WooCommerce\Blueprint\UseWPFunctions;
+
+class ImportSetSiteOptions implements StepProcessor {
+	use UseWPFunctions;
+	public function process( $schema ): StepProcessorResult {
+		$result = StepProcessorResult::success( SetSiteOptions::get_step_name() );
+		foreach ( $schema->options as $key => $value ) {
+			if ( is_object( $value ) ) {
+				$value = (array) $value;
+			}
+			$updated = $this->wp_update_option( $key, $value );
+			$updated && $result->add_info( "{$key} has been updated" );
+			if ( ! $updated ) {
+				$current_value = $this->wp_get_option( $key );
+				if ( $current_value === $value ) {
+					$result->add_info( "{$key} has not been updated because the current value is already up to date." );
+				}
```

Comment:
Should we add a log when the current value is not equal to the new value?

Reply:
I think that's already done in https://github.com/woocommerce/woocommerce/pull/49763/files/2640d3f8bda273d5acc2a46c5c07c5228379cd72#diff-e7ab519e715d22debbce0403824f1c0197afc2b5b2517858beb4dcf79fab10d9R19 

Value gets updated when the current value is different than the new value AFAIK.

As per Ilyas' suggestion, I've changed `$updated && $result->add_info(...)..` to a full if statement: [Use full if statement](https://github.com/woocommerce/woocommerce/pull/49763/commits/622376c77688ccb61188b57c16210b75f786ca14)